quadrilaterals.py

```diff
@@ -0,0 +1,85 @@
+#draw a quarter of a square (symmetry coming)
+import random
+
+class Quadrilateral:
+    def __init__(self, x1, x2, y1, y2, col):
+        '''
+        Attributes:
+        self.xTL x value of the top left
+        self.xBL x value of the bottom left
+        self.xTR x value of the top right
+        self.xBR x value of the bottom right
+        self.yTL y value of the top left
+        self.yTR y value of the bottom left
+        self.yBL y value of the top right
+        self.yBR y value of the bottom right
+        self.col color
+        '''
+        self.xTL = x1
+        self.xBL = x1
+        self.xTR = x2
+        self.xBR = x2
+        self.yTL = y1
+        self.yTR = y1
+        self.yBL = y2
+        self.yBR = y2
+        self.col = col
+
+
+        
+
+def draw_quad(quad):
+##    M = moveto
+##    L = lineto
+    return f'<path d="M{quad.xTL} {quad.yTL} \
+            L{quad.xTR} {quad.yTR} \
+            L{quad.xBR} {quad.yBR} \
+            L{quad.xBL} {quad.yBL}Z" stroke="black" fill="{quad.col}"/>'
+
+def make_quadrilaterals(num, imageSize, colors = ('#044BD9', '#0583F2', '#05AFF2', '#05DBF2', '#fa7f70')):
+    header = f'<svg viewBox="0 0 {imageSize} {imageSize}" xmlns="http://www.w3.org/2000/svg">\n'
+    grid = {}
+    quad_size = imageSize/num
+    print(header)
+    for r in range(num):
+        for c in range(num):
+            q = None
+            q = Quadrilateral(
+                            c * quad_size, c * quad_size + quad_size, r * quad_size, 
+                            r * quad_size + quad_size, random.choice(colors))
+            grid[(r, c)] = q
+##            print(r,c,q.xTL,q.yTL, q.xBR,q.yBR)
+    #shifting stuff
+    for r in range(1, num):
+        for c in range(1, num):
+            diffx = random.randint(0, quad_size // 2) - quad_size // 4                             
```

Comment:
neat. As a suggestion, maybe try Perlin noise so that the skew is nicely distributed over the picture.